frontend/components/mobile/MobileSearchInput.tsx

```diff
@@ -84,34 +117,38 @@ export default function MobileSearchInput(props: MobileSearchInputProps) {
           <input
             id="location_input"
             className="w-36 placeholder:text-2xl text-2xl border-b-2 border-gray-400 outline-none rounded-2xl"
-            placeholder="Location"
-            value={searchLocation}
+            placeholder="Sydney"
+            value={location}
             onChange={(event) => {
-              setSearchLocation(event.target.value);
+              setLocation(event.target.value);
             }}
+            onKeyDown={handleKeyPress}
           />
         </div>
-        <div className="w-full mt-14 ml-1">
+        {/* <div className="w-full mt-14 ml-1">
```

Comment:
remove this

Reply:
We want to implement tags in the future still, so I think we can leave it there